backoffice/templates/backoffice/courses/list.html

```diff
@@ -49,34 +49,31 @@ <h1>{% trans "Courses" %}</h1>
                     <th>{% trans 'End' %}</th>
                     <th>{% trans 'Enrollment start' %}</th>
                     <th>{% trans 'Enrollment end' %}</th>
-                    <th>{% trans 'Enrollments' %}</th>
-                    <th>{% trans 'Mode' %}</th>
```

Comment:
As discussed IRL these 2 fields are very useful and should be kept.

We can improve performance by caching them. For example, declare them as a method property on the model in which you check if the course enrollment is open or closed: 
- If course enrollment is open, try to get the field value from cache or compute it and cache it for a random period between 1h and 2'h (random so that we don't recompute it on too many courses on a given request). 
- If course enrollment is closed, try to get the field value from cache or compute it and cache it for a random period between 1 week and 2 weeks.